randovania/games/prime1/logic_database/Phazon Mines.txt

```diff
@@ -887,7 +887,7 @@ Extra - aabb: [-132.15839, 87.526505, -111.61871, -26.354706, 153.05708, -8.0265
               All of the following:
                   After Plasma Processing Item
                   Any of the following:
-                      Combat (Intermediate) or Normal Damage ≥ 200
+                      Charge Beam or Combat (Intermediate) or Normal Damage ≥ 200
```

Comment:
this would allow you to just have charge beam but not all beams and still pass by, likely should be in it's own statement (charge or combat).

Reply:
Also need to update changelog on this, forgot